conda/meta.yaml

```diff
@@ -12,8 +12,7 @@ build:
 
 test:
     requires:
-        - ts-conda-build =0.3
-        - ts-dds

```

Comment:
Is this supposed to be removed?

Reply:
Yeah it can be removed since ts-salobj has a dependency on ts-dds already and would be pulled in regardless of being there or not.